### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by LoviMadaan**
### In this branch
- I have added updated html and css file.
- I have added cards.
- I have added contact me details.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,5 +61,203 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
           </div>
           </div>
       </section>
+      <section class="card-works">
+        <div class="card">
+          <div class="card-img">
+            <img src="./image/Snapshoot Portfolio.svg" alt="img">
```
Comment (IngDavidTM) [inline index.html:67]: - [ ] While you did a good job creating all the required cards :+1: there are some things that you missed about the design.
  - The image should use `border-radius` in each corner :+1: 
  - The elements of the list should be gray like the template :+1: 
  - Each technology should have different font color and background just like the Figma design :+1: 
  - All the buttons inside your project should follow the required design that I mentioned above.

**Your design**

WhatsApp Image 2023-01-14 at 8 58 21 AM

**Figma template**

Screenshot from 2023-01-14 09-01-09

File: index.html [hunk 2]
```diff
@@ -61,5 +61,203 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+              <div class="skill-container">
+                <button class="btn-chev" type="button">
+                  <img src="./image/Frame 9 (1).svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Skills -->
+          </ul>
+        </section>
```
Comment (IngDavidTM) [inline index.html:232]: - [ ] You did a good job creating the last part of your project :+1: :100: , but as a project requirement you should try to make your design as much as possible to the Figma template.
 - The background should be white and have a border radius on the right upper corner
 - Languages, `frameworks` and `skills` shouldn't have borders and should have `v` icons.

**Your design**

Screenshot from 2023-01-14 09-07-09

**Figma design**

Screenshot from 2023-01-14 09-07-39

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @LoviMadaan  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -61,5 +61,203 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+                <li class="description"><span></span>Back End Dev</li>
+                <li class="description"><span></span>2015</li>
+              </ul>
+              <p class="card-para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <ul class="card-langs">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="card-btn">See Project</button><button type="submit"></button>
```
[the remark this answers is not in the document]
Comment (LoviMadaan) [reply index.html:143]: I don't know how to add that. please guide me

[the remark this answers is not in the document]
Comment (IngDavidTM) [reply index.html]: Hi @LoviMadaan [here](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) you can find some examples of how to add effects to the buttons

File: index.html [hunk 4]
```diff
@@ -61,5 +61,241 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+             </li>
+            </ul>
+            <button class="btn-default" type="button">Get my resume</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="images/chev-dow.svg" alt="" />
```
Comment (RaoAkif) [inline index.html:229]: I think you have put a down arrow icon here. But this icon `images/chev-dow.svg` is missing. It doesn't exist in the folder. Kindly add that arrow.

File: style.css [hunk 5]
```diff
@@ -117,3 +117,302 @@ ul {
...
+  align-items: center;
+  padding: 1rem;
+  border: 1px solid #6070ff;
+  color: #6070ff;
+  border-radius: 1rem;
+  width: 40%;
+}
+
+.card-btn:hover {
+  background: #6070ff;
```
Comment (RaoAkif) [inline style.css:234]: According to FIGMA design, this color on press is `#2230D2`. Kindly apply them on all the button classes, like  `.card-btn`, `btn-default`, etc.

``` 
.btn-default:active {
  background: #2230D2;
  color: #fff;
}
```

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @LoviMadaan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -61,5 +61,241 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+            <ul class="tags">
+              <li class="tag">html</li>
+              <li class="tag">css</li>
+              <li class="tag">javaScript</li>
+            </ul>
+            <div class="action">
+              <button class="btn-default" type="button">See Project</button>
+            </div>
+          </div>
+        </article>
```
Comment (IngDavidTM) [inline index.html:94]: - [ ] Good job with implementing all the required changes :+1:, but as you can see below there is no difference between all the cards. It means that the background should be gray and each card should have border :100: 

**Your design**

Screenshot from 2023-01-14 12-41-07

**Figma design**

Screenshot from 2023-01-14 12-41-44

File: index.html [hunk 7]
```diff
@@ -61,5 +61,241 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+                  JavaScript
+                </li>
+                <li>
+                  <img src="./image/Frame 10.svg" alt="HTML" />
+                  HTML
+                </li>
+                <li>
+                  <img src="./image/Frame 11.svg" alt="CSS" />
+                  CSS
+                </li>
```
Comment (IngDavidTM) [inline index.html:245]: - [ ] Good job implementing the `<li>` tags here :+1:, but I have noticed that your design is not matching the required Figma template. It has two names for each element as you can see bellow

**Your design**

Screenshot from 2023-01-14 12-33-11

**Figma template**

Screenshot from 2023-01-14 08-37-05

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @LoviMadaan  🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (LoviMadaan) [reply index.html:245]: I not getting this comment. Please specify the change you want me to implement.

File: index.html (hunk 6 above)
Comment (LoviMadaan) [reply index.html:94]: Where should i implement change. Please specify.

Comment (IngDavidTM) [reply index.html:94]: As you can see in the above images the grey background should be outside each work card and also each card should have a border

Comment (IngDavidTM) [reply index.html:94]: It will help you to differentiate each of them :+1:

File: index.html [hunk 8]
```diff
@@ -61,5 +61,238 @@ <h2 class ="contact-link">LET’S CONNECT</h2>
...
+                <span class="seprator-circle"></span>
+                <p class="role">Back End Dev</p>
+                <span class="seprator-circle"></span>
+                <p class="year">2015</p>
+              </div>
+            </div>
+            <p class="project-details">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
```
Comment (algerina) [inline index.html:84]: - [ ] You did great work by adding a border to the cards but still, the cards and the background have the same color; to fix the issue as required by the previous reviewer just add a grey `background-color` to the `works` class in your CSS file, consider also adding a little `margin-top` to each `card` to match the Figma design.

**Review by algerina — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

### Hello @LoviMadaan  ,
 ✋🏼 Nice done I am impressed. Your project is great. There are just some small changes to add.

HIGHLIGHTS
- Your Project looks great :heavy_check_mark: 
- No Linter errors :heavy_check_mark: 
- You have a descriptive PR :heavy_check_mark:   

## Required changes :

- [ ] Please, check the comments under the review.

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by Owusu-Desmond — APPROVED** [review]
Hi @LoviMadaan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**